### DISCUSSION TITLE
Having issue installing LazyVIm

### DISCUSSION THREAD

**Discussion by uldrim** [LazyVim/LazyVim#965] [Q&A] [answered]
After cloning the repo and starting the nvim
I am getting nvim-treesitter error: Error during compilation.
Any advice?
![Screenshot from 2023-06-23 16-44-03](https://github.com/LazyVim/LazyVim/assets/45782936/6c9a5ff6-7178-4892-bc30-fb79fbeb4759)

**Comment by folke** [comment, marked as answer]
For treesitter you need a proper c compiler. See here: https://github.com/nvim-treesitter/nvim-treesitter#requirements.

I'll update the LazyVim requirements to reflect this

**Reply by uldrim** [reply]
Oh, thank you so much.
